src/app/services/deposits/deposits.service.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { TestBed } from '@angular/core/testing';
+
+import { DepositsService } from './deposits.service';
```

Comment:
hey @Genza999, we should use absolute imports

Reply:
Thanks @CryceTruly , I have worked on this